zerver/tests/test_push_notifications.py

```diff
@@ -923,13 +925,23 @@ def test_send_remove_notifications_to_bouncer(self) -> None:
 
         with self.settings(PUSH_NOTIFICATION_BOUNCER_URL=True), \
                 mock.patch('zerver.lib.push_notifications'
-                           '.send_notifications_to_bouncer') as mock_send_android, \
+                           '.send_notifications_to_bouncer') as mock_send, \
                 mock.patch('zerver.lib.push_notifications.get_base_payload',
                            return_value={'gcm': True}):
             handle_remove_push_notification(user_profile.id, [message.id])
-            mock_send_android.assert_called_with(
+            mock_send.assert_called_with(
                 user_profile.id,
-                {},
+                {
+                    'badge': 0,
+                    'custom': {
+                        'zulip': {
+                            'gcm': True,
+                            'event': 'remove',
+                            'zulip_message_ids': str(message.id),
+                            'zulip_message_id': message.id,
```

Comment:
@gnprice do we need the two lists hack for remove notifications here?  I would imagine that the Apple client code will never have a version that looks at `zulip_message_id` and not `zulip_message_ids`.

(And do we want to just call it `mesasge_ids`?  I think that's a better name)

Reply:
Oh, in fact, the `zulip_message_id` here is actually a bug in the test, caused by reuse of the mocked `get_base_payload` response dictionary that I just complained about above.  So that was never intentional or even present in the real API!

I think maybe `zulip_message_ids` makes sense for compatibility with the Android naming, after a bit more thought?

Reply:
> I think maybe `zulip_message_ids` makes sense for compatibility with the Android naming, after a bit more thought?

Yeah, I think so.